plugin/trino-mariadb/src/test/java/io/trino/plugin/mariadb/BaseMariaDbTableIndexStatisticsTest.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.mariadb;
+
+import io.trino.testing.MaterializedRow;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+import static java.lang.String.format;
+
+public abstract class BaseMariaDbTableIndexStatisticsTest
+        extends BaseMariaDbTableStatisticsTest
+{
+    protected BaseMariaDbTableIndexStatisticsTest(String dockerImageName)
+    {
+        super(
+                dockerImageName,
+                nullFraction -> 0.1, // Without mysql.column_stats we have no way of knowing real null fraction, 10% is just a "wild guess"
+                varcharNdv -> null); // Without mysql.column_stats we don't know cardinality for varchar columns
```

Comment:
`mysql` -> `mariadb`?

Reply:
no, that's how this table is called in MariaDB